airflow-core/src/airflow/ui/src/queries/useConnectionTypeMeta.ts

```diff
@@ -99,9 +99,7 @@ export const useConnectionTypeMeta = () => {
     );
 
     if (populatedStandardFields.url_schema) {
-      if (!populatedStandardFields.schema) {
-        populatedStandardFields.schema = populatedStandardFields.url_schema;
-      }
+      populatedStandardFields.schema = populatedStandardFields.url_schema;
```

Comment:
I think the logic is not exactly equivalent. `populatedStandardFields.schema ??= populatedStandardFields.url_schema`